endpointmanager/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -195,7 +195,8 @@ func ReceiveCapabilityStatements(ctx context.Context,
 	}
 
 	errs := make(chan error)
-	go messageQueue.ProcessMessages(messages, saveMsgInDB, &args, errs)
+	ctx, _ = context.WithCancel(ctx)
```

Comment:
since you're not using `cancel`, i think you can just pass in the original `ctx`